### PR TITLE
Sidebar images for territories

### PR DESCRIPTION
**Pull request by mgwalker**
#### American Samoa

outline of American Samoa

#### Guam

outline of Guam

#### Commonwealth of Northern Mariana Islands

outline of Commonwealth of Northern Mariana Islands

#### US Virgin Islands

outline of US Virgin Islands

### This pull request changes...

- adds images for American Samoa, Guam, the Commonwealth of the Northern Mariana Islands, and the US Virgin Islands
- address #1758

### This pull request is ready to merge when...

- [x] Tests have been updated (and all tests are passing)
- [x] This code has been reviewed by someone other than the original author
- [ ] The experience passes a basic manual accessibility audit (keyboard nav, screenreader, text scaling) OR an exemption is documented
- [ ] The change has been documented
  - [ ] Associated OpenAPI documentation has been updated
- [x] Changelog is updated as appropriate

### This feature is done when...

- [ ] Design has approved the experience
- [x] Product has approved the experience

### CODE REVIEW

**Comment by mgwalker** [comment]
Aside from Guam, the territories are made of relatively small islands spread over relatively large areas, so when scaled down to fit into the sidebar, they're hard to see. I tried to make this better by taking out some of the empty ocean between the islands, but they're still kinda not great.

I reached out to the design chapter at 18F for thoughts on ways to make them more visually meaningful, and one idea was to switch from geographies to flags. All the states, territories, and districts have flags! Those are not only more recognizable for the territories, but possibly also for some of the states that are essentially just rectangles.

Just something to think about.

**Comment by mgwalker** [comment]
Maybe request for comments/thoughts from @jeromeleecms and @lauraponce? This also seems like SUPER low priority at the moment, but it was out there lingering and it was a good distraction from putting together slides for a few minutes... 😂

**Comment by jeromeleecms** [comment]
No worries - small tasks when working on slides can be a welcome mental break... flags could def. be on the table for a future enhancement...

File: web/scripts/stateIcons.js [hunk 1]
```diff
@@ -26,7 +26,7 @@ Object.keys(data).forEach(id => {
   const state = data[id];
   const geo = feature(state, state.objects.icon);
 
-  const dest = path.join(__dirname, `../src/static/img/${id}.svg`);
+  const dest = path.join(__dirname, `../src/static/img/states/${id}.svg`);
```
Comment (hbillings) [inline web/scripts/stateIcons.js]: Good call on moving these to their own img folder

**Pull request merged**